### PR TITLE
Changes in Jenkins file

### PR DESCRIPTION
**Pull request by shyadav10**
Link to the kraken run - 
https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/shyadav-e2e-benchmarking-multibranch-pipeline/job/kraken/48/

### CODE REVIEW

File: set_kraken_files.sh [hunk 1]
```diff
@@ -12,6 +12,18 @@ cd ${KRAKEN_SCENARIO}
 ls
 
 source env.sh
+
+# if [[ ${KRAKEN_SCENARIO} == "node-cpu-hog" ]]; then
```
Comment (paigerube14) [inline set_kraken_files.sh:16]: if we aren't using these, lets go ahead and remove

File: Jenkinsfile [hunk 2]
```diff
@@ -15,7 +15,28 @@ pipeline {
         string(name: 'PAUSE_TIME', defaultValue: '4', description: 'Number of minutes to pause.')
         string(name: 'ITERATIONS', defaultValue: '1', description: 'Number of iterations to run of the chaos scenario.')
         choice(choices: ["application-outages","container-scenarios","namespace-scenarios","network-chaos","node-scenarios","pod-scenarios","node-cpu-hog","node-io-hog", "node-memory-hog", "power-outages","pvc-scenario","time-scenarios","zone-outages"], name: 'KRAKEN_SCENARIO', description: '''Type of kraken scenario to run''')
-        
+        string(name: 'NODE_CPU_PERCENTAGE', defaultValue: "", description: 'int the CPU load by percentage')
```
Comment (paigerube14) [inline Jenkinsfile:18]: lets update the formatting of this variable and the ones above to be like the below, with each name/desc/defaultValue on a new line

Comment (paigerube14) [reply Jenkinsfile:18]: and lets also align every "string" to be the same indentation

**Review by shahsahil264 — APPROVED** [review]
LGTM

**Pull request merged**